public/locales/en/labelmap.json

```diff
@@ -0,0 +1,13 @@
+{
+    "is-correct": "Is this label correct?",
+    "click-here": "Click here",
+    "explore-street": "to explore the street!",
+    "agree": "Agree",
+    "disagree": "Disagree",
+    "not-sure": "Not Sure",
+    "label-type": "Label Type",
+    "validations": "Validations",
+    "add-comment": "Add comment here...",
+    "submit": "Submit comment",
+    "comment-submitted" : "Comment Submitted"
+}
```

Comment:
Make sure to add a newline at the end of files! A la [style guide](https://github.com/ProjectSidewalk/SidewalkWebpage/wiki/Style-Guide)